pkg/controller/installation/core_controller.go

```diff
@@ -896,7 +845,7 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 			return reconcile.Result{}, err
 		}
 		for _, x := range crb.Finalizers {
-			if x == render.NodeFinalizer {
```

Comment:
This was an incidental name change as a result of my investigation into some uninstall errors I was seeing with this change. I think this is a better name anyway, since it's the CNI resources we're actually finalizing.